modules/library/cql/src/test/java/org/geotools/filter/text/ecql/ECQLGeoOperationTest.java

```diff
@@ -154,43 +161,42 @@ public void functionAsFirstArgument() throws CQLException {
         
         Filter resultFilter = CompilerUtil.parseFilter(language,"INTERSECTS(centroid(the_geom), POINT(1 2))");
 
-        Assert.assertTrue("Intersects was expected", resultFilter instanceof Intersects);
+        assertTrue("Intersects was expected", resultFilter instanceof Intersects);
     }
 
     @Test
     public void functionAsSecondArgument() throws CQLException {
         
         Filter resultFilter = CompilerUtil.parseFilter(language,"INTERSECTS(the_geom, buffer(POINT(1 2),10))");
         
-        Assert.assertTrue("Intersects was expected", resultFilter instanceof Intersects);
+        assertTrue("Intersects was expected", resultFilter instanceof Intersects);
 
         
         resultFilter = CompilerUtil.parseFilter(language,"INTERSECTS(the_geom, buffer(the_geom,10))"); 
         
-        Assert.assertTrue("Intersects was expected", resultFilter instanceof Intersects);
+        assertTrue("Intersects was expected", resultFilter instanceof Intersects);
     }
 
     @Test
     public void functionAsFirstAndSecondArgument() throws CQLException {
         
         Filter resultFilter = CompilerUtil.parseFilter(language,"INTERSECTS(centroid(the_geom), buffer(POINT(1 2) ,10))");
 
-        Assert.assertTrue("Intersects was expected", resultFilter instanceof Intersects);
+        assertTrue("Intersects was expected", resultFilter instanceof Intersects);
 
     }
     
-    /**
-     * This test require an extension of bbox filter
-     */
-    @Ignore
-    public void bboxWithFunctionOnFirstArgument(){
-        
-        //  TODO function on first arg BBOX( buffer( the_geom , 10), 10,20,30,40 )
-        //  TODO function on second argument BBOX( buffer( the_geom , 10), buffer( POINT( 15,15), 10) )
-        //  TODO Geometry on first and second arguments BBOX( POLYGON( .... ), POLYGON( .... ), 10) )
-        //  TODO Proposal in geotools wiki for this
+    @Test
+    public void IntersectsWithReferencedGeometry() throws CQLException, FactoryException {
```

Comment:
Method name should start with lowercase "i".

Reply:
Whoops (copy paste, many other methods starting with a capital in that class)